script.js

```diff
@@ -1 +1,189 @@
+  }
+  todoContent.className = 'todoContent';
+  todoContent.htmlFor = `todoCheckbox${checkboxNumber++}`;
+  todoContent.textContent = todo;
+  todoDelete.className = 'todoDelete';
+  todoDelete.textContent = 'X';
+
+  todoCard.append(checkbox, todoContent, todoDelete);
+  if (isDone) {
+    document.querySelector('.doneSection').append(todoCard);
+  } else {
+    document.querySelector('.todoSection').append(todoCard);
+  }
+};
+
+//localstorage에 저장된 todo,done 불러오기
+const todoList = JSON.parse(localStorage.getItem('todo'));
+const donelist = JSON.parse(localStorage.getItem('done'));
+if (todoList) {
+  todoList.forEach((todo) => {
+    createTodoCard(todo, false);
+  });
+}
+if (donelist) {
+  donelist.forEach((todo) => {
+    createTodoCard(todo, true);
+  });
+}
+
+//localstorage에 todo,done 추가하기
+const addTodoToLocalStorage = (todo) => {
+  const todoList = JSON.parse(localStorage.getItem('todo'));
+  localStorage.removeItem('todo');
+  if (todoList) {
+    const newTodoList = [...todoList, todo];
+    localStorage.setItem('todo', JSON.stringify(newTodoList));
+  } else {
+    localStorage.setItem('todo', JSON.stringify([todo]));
+  }
+};
+const addDoneToLocalStorage = (todo) => {
+  const doneList = JSON.parse(localStorage.getItem('done'));
+  localStorage.removeItem('done');
+  if (doneList) {
+    const newDoneList = [...doneList, todo];
+    localStorage.setItem('done', JSON.stringify(newDoneList));
+  } else {
+    localStorage.setItem('done', JSON.stringify([todo]));
+  }
+};
+
+//localstorage에 todo,done 삭제하기
+const deleteTodoFromLocalStorage = (todo) => {
+  const todoList = JSON.parse(localStorage.getItem('todo'));
+  const newTodoList = todoList.filter((item) => item !== todo);
+  localStorage.setItem('todo', JSON.stringify(newTodoList));
+};
+const deleteDoneFromLocalStorage = (todo) => {
+  const doneList = JSON.parse(localStorage.getItem('done'));
+  const newDoneList = doneList.filter((item) => item !== todo);
+  localStorage.setItem('done', JSON.stringify(newDoneList));
+};
+
+/* 
+  doneCount: 완료 된 todo 개수
+  todoCount: 등록 된 todo 개수
+*/
+
+let doneCount = document.querySelectorAll('.doneCard').length;
+let todoCount = document.querySelectorAll('.todoCard').length + doneCount;
+
+//등록 된 todo, 완료 된 todo 개수 표시
+const noteTodoProgress = () => {
+  document.querySelector(
+    '.todoProgress'
+  ).textContent = `${doneCount} 개 / ${todoCount} 개 `;
+};
+noteTodoProgress();
+
+//오늘 날짜 표시
+const today = new Date();
+const year = today.getFullYear();
+const month = today.getMonth() + 1;
+const date = today.getDate();
+const day = today.getDay();
+const dayList = [
+  'Sunday',
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday',
+];
+document.querySelector('.headerDate').append(`${date}`);
+document.querySelector('.headerMonth').append(`${month}`);
+document.querySelector('.headerYear').append(`${year}`);
+document.querySelector('.headerDay').append(`${dayList[day]}`);
```

Comment:
오늘 날짜 표시하는 로직이 조금 긴 것 같습니다!!

```suggestion
const today = new Date();
const dayList = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];

function updateHeader(selector, content) {
  document.querySelector(selector).append(content);
}

updateHeader('.headerDate', today.getDate());
updateHeader('.headerMonth', today.getMonth() + 1);
updateHeader('.headerYear', today.getFullYear());
updateHeader('.headerDay', dayList[today.getDay()]);
```
document.querySelector(...).append(...) 호출을 줄이고,
querySelector와 append를 결합하는 호출을 하나의 함수로 만들어 재사용하면 더 가독성이 좋아질 것 같습니다 !!